scripts/sync-gsuite.js

```diff
@@ -0,0 +1,233 @@
+
+// PART 1: get all gsuite under /individuals, and for each one make a query to core
```

Comment:
Might be good to make a query similar to this which will add the GSuite accounts of locals to MyAEGEE and afterwards check which GSuite accounts in all organizational units are not linked to MyAEGEE and what we want to do with those

Reply:
it is possible to query different OUs so this is possible to implement for `/antennae`